### PR TITLE
added rudimentary front-end for sign-up form. Next step will be hooki…

### PR DESCRIPTION
**Pull request by kevinwu127**
…ng up firebase

### CODE REVIEW

File: src/pages/signup/signup.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    });
+  }
+
+
+  dismiss() {
+    this.viewCtrl.dismiss();
+  }
+
+  logForm() {
+    alert("You've signed up for Trump Supporters Daily! Thank you for your continued support of the United States of Shitters.");
```
Comment (richardsun0713) [inline src/pages/signup/signup.ts]: Let's hope we remember to change this

Comment (kevinwu127) [reply src/pages/signup/signup.ts]: Hey man, that's the beauty of software engineering.

File: src/pages/signup/signup.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import { Component } from '@angular/core';
+
+import { ViewController, NavController } from 'ionic-angular';
```
Comment (richardsun0713) [inline src/pages/signup/signup.ts]: NavController unused

File: src/pages/home/home.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import { Component } from '@angular/core';
+
+import { ModalController, NavController, NavParams } from 'ionic-angular';
```
Comment (richardsun0713) [inline src/pages/home/home.ts]: NavParams unused

**Comment by kevinwu127** [comment]
K gna merge

**Pull request merged**